Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Isotropic_remeshing/remesh_impl.h

```diff
@@ -906,8 +915,9 @@ namespace internal {
         put(degree, vc, vvc);
         put(degree, vd, vvd);
 
+#ifdef CGAL_PMP_REMESHING_VERBOSE
         ++nb_flips;
-
+#endif
 #ifdef CGAL_PMP_REMESHING_VERBOSE_PROGRESS
         std::cout << "\r\t(" << nb_flips << " flips)";
```

Comment:
why not use the existing macro? `nb_flips` and `nb_collapses` are defined behind `CGAL_PMP_REMESHING_VERBOSE` and used behind `CGAL_PMP_REMESHING_VERBOSE_PROGRESS`.

Reply:
I was wondering it made sense to also write the lower case versions, that is functions.

Reply:
> The more complicated macros call `#define CGAL_PMP_REMESHING_VERBOSE`

That's not what I meant; I thought it was only using within the `_PROGRESS` block but I hadn't seen there is a call at the end:
```
#ifdef CGAL_PMP_REMESHING_VERBOSE
      std::cout << "\r\tdone ("<< nb_flips << " flips)" << std::endl;
#endif
```
Sorry!